app/src/main/java/org/hwyl/sexytopo/control/Log.java

```diff
@@ -71,6 +73,10 @@ public static synchronized void device(String message) {
         android.util.Log.i(SexyTopo.TAG, message);
     }
 
+    public static String formatDebugString(String value) {
```

Comment:
There is possibly a more idiomatic way to debug, print a string via a logger; but I'm not very familiar with Java or the Android ecosystem.

Reply:
Not that I can think of, although haven't written much in Java for years (apart from SexyTopo - and should probably change it to Kotlin at some point).